src/pika_server.cc

```diff
@@ -1487,6 +1490,46 @@ void PikaServer::AutoKeepAliveRSync() {
   }
 }
 
+void PikaServer::AutoResumeDB() {
+  struct statfs disk_info;
+  int ret = statfs(g_pika_conf->db_path().c_str(), &disk_info);
```

Comment:
I see it uses `statfs`(deprecated) on the other one. I perfer to use `statvfs` based on https://www.mkssoftware.com/docs/man3/statfs.3.asp

Reply:
@infdahai Could you please help me review the code? #1822 I have made relevant changes to your suggestion